docs/sources/alerting/manage-notifications/template-notifications/using-go-templating-language.md

```diff
@@ -71,6 +71,14 @@ The name of the annotation is {{ .Name }}, and the value is {{ .Value }}
 {{ end }}
 ```
 
+## Use a specific label value
+
+You may only want to use some specific label values in your text, or use a longer description. 
+
+```
+This alert is in the folder {{ index .Labels "grafana_folder" }}
```

Comment:
Would be great if we can put this in a range. A lot of users don't look at the context of code examples and instead just use them verbatim and then complain it doesn't work, so adding a `range` here should make it work if copied without reading the rest of the page.

```suggestion
{{ range .Alerts }}
This alert is in the folder {{ index .Labels "grafana_folder" }}
{{ end }}
```